docs/glassfish/publish/pom.xml

```diff
@@ -0,0 +1,239 @@
+                                            <groupId>org.glassfish.docs</groupId>
+                                            <artifactId>distribution</artifactId>
+                                            <version>${project.version}</version>
+                                            <outputDirectory>
+                                                ${dir.snap}
+                                            </outputDirectory>
+                                        </artifactItem>
+                                    </artifactItems>
+                                </configuration>
+                            </execution>
+                            <!--
+                                Get the collection of documentation for the
+                                5.1.0 release.
+                            -->
+                            <execution>
+                                <id>get-510</id>
+                                <phase>generate-sources</phase>
+                                <goals>
+                                    <goal>unpack</goal>
+                                </goals>
+                                <configuration>
+                                    <includeGroupIds>
+                                        org.glassfish.docs
+                                    </includeGroupIds>
+                                    <excludes>META-INF/**</excludes>
+                                    <artifactItems>
+                                        <artifactItem>
+                                            <groupId>org.glassfish.docs</groupId>
+                                            <artifactId>distribution</artifactId>
+                                            <version>5.1.0</version>
+                                            <outputDirectory>
+                                                ${dir.510}
+                                            </outputDirectory>
+                                        </artifactItem>
+                                    </artifactItems>
+                                </configuration>
+                            </execution>
+                            <!--
+                                Add additional execution blocks here as
+                                new releases are made, using the above
+                                as a template.  Don't forget to update
+                                the docs.version property.
+                            -->
+                        </executions>
+                    </plugin>
+
+                    <!--
+                        Create a symlink from "latest" to the most recently
+                        released version of the docs.
+                    -->
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <version>1.8</version>
```

Comment:
Plugin version should be managed in the parent pom

Reply:
Moved version to top level pom.